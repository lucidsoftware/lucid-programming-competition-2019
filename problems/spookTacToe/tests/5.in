7 7
.o###.#
..oo...
......#
.......
....#..
...#..#
x##...o