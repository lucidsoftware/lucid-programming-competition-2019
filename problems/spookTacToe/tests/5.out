oo###o#
xxooxxo
ooxxoo#
xxooxxo
ooxx#oo
xoo#xx#
x##xxoo